parlai/scripts/eval_model.py

```diff
@@ -32,6 +33,15 @@ def setup_args(parser=None):
         parser = ParlaiParser(True, True, 'Evaluate a model')
     parser.add_pytorch_datateacher_args()
     # Get command line arguments
+    parser.add_argument(
+        '-rf',
```

Comment:
What do you think about having this parameter empty by default?

Consider if runs an eval_model on a zoo file. We're going to be touching that directory, even in unit tests. I'd rather not have it smash the directories *by default*.

Reply:
i think i agree with this, though one way to fix unit tests is to add this arg to `testing_utils.eval_model`

Reply:
ok, i'm not sure, i kind of liked how it saved if you forget, but i see your point.
i'm wondering if there's a file there if it could append to the report (like the training stats log), possibly with a timestamp
but for now, i've made it empty as requested

Reply:
perhaps if we made the default a `/tmp/` location we would not run into this issue